### DISCUSSION TITLE
parallel route?

### DISCUSSION THREAD

**Discussion by stigmat4j** [vikejs/vike#975] [Discussion] [answered]
<img width="1069" alt="image" src="https://github.com/brillout/vite-plugin-ssr/assets/3407889/c405aa33-a00a-4e59-84fa-f177c0a43504">

Maybe it is implemented or planned?

**Comment by brillout** [comment, marked as answer]
I believe you mean https://vite-plugin-ssr.com/layouts#nested-layouts.

**Reply by stigmat4j** [reply]
Thank you!
